tools/nni_cmd/nnictl_utils.py

```diff
@@ -433,3 +436,38 @@ def monitor_experiment(args):
         except Exception as exception:
             print_error(exception)
             exit(1)
+
+
+def parse_trial_data(trial_data):
+    """output: Dict"""
+    hparam = json.loads(trial_data['hyperParameters'][0])['parameters']
+    reward = json.loads(trial_data['finalMetricData'][0]['data'])
+    if isinstance(reward, float):
+        dict_ret = {**hparam, **{'reward': reward}}
+    elif isinstance(reward, dict):
+        dict_ret = {**hparam, **reward}
+    else:
+        raise ValueError("Invalid finalMetricsData format: {}".format(trial_data['finalMetricData'][0]['data']))
+    return dict_ret
+
+def export_experiment_2csv(args):
+    """export experiment metadata to csv
+    :param: csv file path to export
+    """
```

Comment:
it would be better to use the following format:
"""export experiment metadata to csv
    
    Parameters
    ----------
    args: string
        csv file path to export

    Returns
    -------
    null
    """